### DISCUSSION TITLE
Port : docs.anycable.io / README

### DISCUSSION THREAD

**Discussion by benbonnet** [anycable/anycable#122] [General]
Hello;

just stepping in; providing my first impression that maybe would help for future users if explained / fixed. About the port.

On docs.anycable.io (rails integration), the setup is advertised as the following : 
- config.action_cable.url = "ws://localhost:8080/cable"
- anycable-go --host=localhost --port=3334

On the README here, the setup is advertised as the following :
- config.action_cable.url = "ws://localhost:3334/cable"
- anycable-go --host=localhost --port=3334

Landed first on docs.anycable.io .  
While the latter seems obvious in the end, have to say that it took me quite a while to make sure it was the right way (to be honest, looked through the doc to make sure there wasn't some kind of proxy done by rails via the gem that mapped 8080 to this 3334 port)

Anyhow, looks like there might be a typo somewhere

Regards

**Comment by palkan** [comment]
Thanks!
Replaced all the `3334` with `8080`.
